ruby_oracle/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM ruby:2.3
+MAINTAINER VMIT
```

Comment:
MAINTAINER is deprecated - might want to replace w/ label (https://docs.docker.com/engine/deprecated/#maintainer-in-dockerfile)